src/cli/commands/upgrade-interactive.js

```diff
@@ -57,26 +59,15 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
   const headerPadding = (header, key) =>
     `${reporter.format.bold.underline(header)}${' '.repeat(maxLengthArr[key] - header.length)}`;
 
-  const colorizeName = ({current, wanted}) => (current === wanted ? reporter.format.yellow : reporter.format.red);
+  const colorizeName = (from, to) => reporter.format[colorKeyFromVersions(from, to)];
```

Comment:
I think it'd be better to keep object destructuring here for the sake of named arguments. What do you think?